README.md

```diff
@@ -339,8 +339,10 @@ This is a collection of static analysis tools and code quality checkers. Pull re
 
 * [AppChecker](https://npo-echelon.ru/en/solutions/appchecker.php) :copyright: - Static analysis for C/C++/C#, PHP and Java
 * [Application Inspector](https://www.ptsecurity.com/ww-en/products/ai/) :copyright: - Combined SAST, DAST, IAST security scanner for C#, PHP, Java, SQL languages
+* [AppScan](https://www.ibm.com/support/knowledgecenter/en/SSS9LM_9.0.3/com.ibm.rational.appscansrc.install.doc/topics/system_requirements_language_support.html) :copyright: - Commercial Static Code Analysis. Supports: .NET (C#, ASP.NET, VB.NET) - Microsoft .NET Framework (Versions 2.0, 3.0, 3.5, 4.0, 4.5, and 4.6), ASP (JavaScript/VBScript), C/C++, COBOL, ColdFusion, JavaScript, JavaServer Pages (JSP), Java™ (including support for Android APIs), Perl, PHP (Versions 5.5, 5.6, and 7.0), PL/SQL, T-SQL, Visual Basic 6
```

Comment:
Can we make the description a little shorter? (E.g. by removing the parentheses with the version numbers for .NET and PHP?)